recipients_app/lib/main_dev.dart

```diff
@@ -15,5 +15,5 @@ Future<void> main() async {
   FirebaseFirestore.instance.useFirestoreEmulator('localhost', 8080);
   FirebaseAuth.instance.useEmulator('http://localhost:9099');
 
-  runApp(MyApp());
+  runApp(const MyApp());
```

Comment:
Are all those changes `const` related to the linter checks? Or is there other benefit of making them across whole app?

Reply:
Yes, they are auto fixed by the linter and I will give you some settings if you use VSCode so that they are auto fixed on saving the file. 

const helps to increase performance in dart. const variables (or objects) are already known in compile time and will never change. The compiler then can just reuse this object - because its never going to change - and doesnt need to create a new one.